src/components/select/index.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react'
+import './select.less'
+import { Select } from 'antd';
```

Comment:
Should install [eslint-plugin-import](https://www.npmjs.com/package/eslint-plugin-import), relative import should place after library import